lib/manageiq/appliance_console/prompts.rb

```diff
@@ -160,10 +162,11 @@ def ask_for_disk(disk_name, verify = true)
           q.choice("Don't partition the disk") { nil }
         end
       end
-
-      if verify && disk.nil?
-        say ""
-        raise MiqSignalError unless are_you_sure?(" you don't want to partition the #{disk_name}")
+      unless silent
```

Comment:
We already have a guard here (verify) I don't think we also need to add this conditional.

It wouldn't make very much sense to pass verify true with silent also true, would it?

Reply:
It wouldn't. Thanks